cmd/syft/internal/test/integration/test-fixtures/image-vertical-package-dups/Dockerfile

```diff
@@ -1,6 +1,27 @@
-FROM centos:7.9.2009@sha256:be65f488b7764ad3638f236b7b515b3678369a5124c47b8d32916d6487418ea4
```

Comment:
this image was so old that I was having a hard time replicating the state, so I updated the test fixture + test